scripts/complete_liquidity_provision.js

```diff
@@ -79,9 +87,12 @@ module.exports = async (callback) => {
     const investmentTargetToken = toErc20Units(argv.investmentTargetToken, targetTokenDecimals)
     const investmentStableToken = toErc20Units(argv.investmentStableToken, stableTokenDecimals)
 
-    assert(argv.fleetSize % 2 == 0, "Fleet size must be a even number for easy deployment script")
+    if (argv.brackets.length > 0) {
```

Comment:
nit-pick: if(argv.brackets) would be shorter.

Reply:
```suggestion
    if (argv.brackets) {
```